packages/cosmic-swingset/go.mod

```diff
@@ -6,23 +6,16 @@ require (
 	github.com/Agoric/agoric-sdk v0.0.0-00010101000000-000000000000 // indirect
```

Comment:
These changes to `go.mod` should be removed from the PR.. I think they're an artifact of our build process. I frequently have to unstage them from git before making the commit to avoid including them.

In general I want `go.mod` to be changed only from PRs that are deliberately update dependencies, in the same way we handle changes to `yarn.lock`.